apps/web-app/pages/api/pretix-create-quota/[slug].ts

```diff
@@ -10,7 +10,7 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
         "Content-Type": "application/json"
     }
 
-    const { ticketAmount, ticketId, variationId1} = req.body
+    const { ticketAmount, ticketId, variationId1 } = req.body
```

Comment:
All variationId's should be inside the request body here and then included in the variations array in the body object that will be send to the pretix API endpoint